examples/doublemach-mpi.py

```diff
@@ -399,12 +414,21 @@ def my_post_step(step, t, dt, state):
     def my_rhs(t, state):
         fluid_state = make_fluid_state(state, gas_model)
         return euler_operator(
-            discr, state=fluid_state, time=t, boundaries=boundaries,
-            gas_model=gas_model
-        ) + make_conserved(dim, q=av_operator(
-            discr, q=state.join(), boundaries=boundaries,
-            boundary_kwargs={"time": t, "gas_model": gas_model}, alpha=alpha,
-            s0=s0, kappa=kappa)
+            discr,
+            state=fluid_state,
+            time=t,
+            boundaries=boundaries,
+            gas_model=gas_model,
+            quadrature_tag=quadrature_tag
+        ) + av_operator(
+            discr,
+            cv=fluid_state.cv,
+            boundaries=boundaries,
+            boundary_kwargs={"time": t, "gas_model": gas_model},
+            alpha=alpha,
+            s0=s0,
+            kappa=kappa,
+            quadrature_tag=quadrature_tag
```

Comment:
I'm not wild about the splayed out style where every argument gets its own line.  Also I don't care enough to make a fuss about it.  In general, I have an aversion to having so much vertical space and so little of the available real-estate used.

Reply:
Compacted it down a bit here: https://github.com/illinois-ceesd/mirgecom/pull/570/commits/78f00f488a1ca49117d2bc529e5c9b5ab955a899